src/controller/AbstractDnssdDiscoveryController.cpp

```diff
@@ -25,6 +25,32 @@
 namespace chip {
 namespace Controller {
 
+bool compareIpAddresses(const size_t sourceNumIPs, const size_t destinationNumIPs, const Inet::IPAddress * source,
```

Comment:
Should probably be called `AddressListsSameExceptOrder` or something like that.

Reply:
OK. I see. I changed function name.